modules/standard/CTypes.chpl

```diff
@@ -205,19 +206,19 @@ module CTypes {
   }
 
   /*
-  This class represents a C array with fixed size.  A variable of type c_array
-  can coerce to a c_ptr with the same element type.  In that event, the
-  pointer will be equivalent to `c_ptrTo(array[0])`.  A c_array behaves
-  similarly to a homogeneous tuple except that its indices start at 0 and it is
-  guaranteed to be stored in contiguous memory.  A c_array variable has value
-  semantics. Declaring one as a function local variable will create the array
-  elements in the function's stack. Assigning or copy initializing will result
-  in copying the elements (vs resulting in two pointers that refer to the same
-  elements).  A `nil` c_array is not representable in Chapel.
+  This type represents a C array with fixed size.  A variable of type
+  ``c_array`` can coerce to a ``c_ptr`` with the same element type.  In that
+  event, the pointer will be equivalent to `c_ptrTo(array[0])`.  A ``c_array``
+  behaves similarly to a homogeneous tuple except that its indices start at 0
+  and it is guaranteed to be stored in contiguous memory.  A ``c_array``
+  variable has value semantics. Declaring one as a function local variable will
+  create the array elements in the function's stack. Assigning or copy
+  initializing will result in copying the elements (vs resulting in two pointers
+  that refer to the same elements).  A ``nil`` ``c_array`` is not representable
+  in Chapel.
   */
-  pragma "c_array record"
-  pragma "default intent is ref if modified"
-  record c_array {
+  pragma "c_array record" pragma
+         "default intent is ref if modified" record c_array {
     /* The array element type */
```

Comment:
With the documentation modifications above, the field declarations here feel a little weird imo.  Maybe it would help to provide examples of how to use them in the docs?

Reply:
I think it's something to do with this type having two fields while c_ptr only has one that bothers me, but it would probably be reasonable to make the adjustment for both, yeah